coupling/services/ParallelTimeIntegrationService.h

```diff
@@ -0,0 +1,231 @@
+// Copyright (C) 2023 Helmut Schmidt University
+// This file is part of the Mamico project. For conditions of distribution
+// and use, please see the copyright notice in Mamico's main folder
+
+#pragma once
```

Comment:
You are mixing include guards and `#pragma once` throughout the repository. This is not directly a problem, just confusing.

Reply:
Yes confusing indeed, but I don't want to change it. I use `#pragma once` on purpose for new files, to avoid include guard name conflicts (we frequently compile with different external codes that we can not / do not want to change), so `#pragma once`  is better. However there is a lot of old code using include guards instead and I don't want to touch it (risk of breaking something that works).